pymc3/tuning/starting.py

```diff
@@ -16,9 +17,8 @@
 
 __all__ = ['find_MAP']
 
-
 def find_MAP(start=None, vars=None, fmin=None,
-             return_raw=False,model=None, *args, **kwargs):
+             return_raw=False, model=None, live_disp=False, callback=None, *args, **kwargs):
```

Comment:
Ah totally forgot, thanks for spotting that.  I changed it now so that if `callback` is given,  it overrides `live_disp` if `live_disp` is set to True.